src/Makefile.am

```diff
@@ -0,0 +1,11 @@
+bin_PROGRAMS = snap-run
+snap_run_SOURCES = main.c seccomp_utils.c utils.c
+snap_run_CFLAGS = \
```

Comment:
Hm, no more `make fmt` and `make syntax-check` ? And will `make test` work as before? It is also dropping the udev releated stuff unless I miss something.